kafka-webview-ui/src/main/java/org/sourcelab/kafka/webview/ui/controller/configuration/cluster/ClusterConfigController.java

```diff
@@ -281,6 +301,46 @@ public String clusterUpdate(
             cluster.setTrustStorePassword(null);
         }
 
+        // If sasl is enabled
+        if (clusterForm.getSasl()) {
```

Comment:
$.02 If you move the stuff up from below the else, you can probably figure out a way to short circuit here and remove a level of conditional nesting. Generally speaking this method stinks, it's big and hard to follow.